app/cron.js

```diff
@@ -0,0 +1,487 @@
+/* eslint-disable security/detect-non-literal-regexp */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable max-classes-per-file */
+const {
+  ConfigManager: { getConfig },
+  Kafka: { Consumer },
+  Logger,
+} = require('@dojot/microservice-sdk');
+const util = require('util');
```

Comment:
I still think it would be be nice if you could separate kafka into another file in the future...